cypress/integration/chi-custom-elements/date-picker.spec.js

```diff
@@ -0,0 +1,79 @@
+/// <reference types="Cypress" />
+
+const clickDate = '11/14/2018';
+const clickDate2 = '01/26/2019';
+const thisMonthName = /November\s*2018/;
+const nextMonthName = /December\s*2018/;
+
+describe('Date picker', function() {
+
+  beforeEach(()=>{
+    cy.visit('tests/custom-elements/date-picker.html');
+    //cy.fixture('chidata.json').as('chidata');
+  });
+
+  it('Clicking on a day emits an event. ', function() {
+
+    const spy = cy.spy();
+
+    cy.get('[data-cy="test-active"]').then((el) => {
+      el.on('chiDateChange', spy);
+    });
+
+    cy.get('[data-cy="test-active"]')
+      .find('chi-date')
+      .should('have.class', 'hydrated')
+      .find(`[data-date="${clickDate}"]`)
+      .click()
+      .then(() => {
+        expect(spy).to.be.called;
+        expect(spy.getCall(0).args[0].detail).to.equal(clickDate);
+      });
+  });
+
+  it('Clicking on next month shows next month. ', function() {
+    cy.get('[data-cy="test-active"]')
+      .find('chi-date')
+      .should('have.class', 'hydrated')
+      .find('.m-datepicker')
+      .should('have.class', '-month-starts-on-thu')
+      .find('.m-datepicker__month')
+      .contains(thisMonthName)
+      .get('[data-cy="test-active"]')
```

Comment:
Shouldn't .get be always chained off of cy? I think this could be divided into 3 parts and every part should start with cy.get...

Reply:
@SergioQCL It can be divided but it is not needed. It does a new start chained in your test. For me this way makes me easier to follow what is a unique test. Tell me if you agree or if not I will change.